### PR TITLE
Registration and dashboard

### PR DESCRIPTION
**Pull request by emilianya**

### CODE REVIEW

File: src/classes/Token/Token.ts [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    }
+
+    toString() {
+        return this.token;
+    }
+
+    static parse(token: string|Token) {
+        // JR.RE.fz_z_MVQj8UQ3lY_UmrUjekE_2U4dn8d4WIkfHqJ7ZQRN-O1BNu6xfKINhKCrv1I.ljlioshm.ljlirzu1
+        // JR.AC.fz_z_MVQj8UQ3lY_UmrUjekE_2U4dn8d4WIkfHqJ7ZQRN-O1BNu6xfKINhKCrv1I.ljlioshm.ljlirzu1
+        // XX.YY.ZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZ.CCCCCCCC.EEEEEEEE
```
Comment (Vukkyy) [inline src/classes/Token/Token.ts:77]: The fuck happened here? Did you fall asleep?

**Comment by Vukkyy** [comment]
OOPS I DIDNT CLICK APPROVE

File: src/classes/Token/Token.ts (hunk 1 above)
Comment (emilianya) [inline src/classes/Token/Token.ts:77]: It explains how the tokens work

**Review by nyakase — APPROVED** [review]

File: src/classes/Token/Token.ts (hunk 1 above)
Comment (nyakase) [reply src/classes/Token/Token.ts:77]: it was a joke ;<

**Pull request merged**